### DISCUSSION TITLE
react-native-sdk + jwt coming from google login

### DISCUSSION THREAD

**Discussion by ealeksandrov7** [Web3Auth/Web3Auth#601] [Help]
I am trying to achieve something similar to https://github.com/orgs/Web3Auth/discussions/547.
I want to do a "custom login" to google, get the idToken and use it to construct a private key using web3auth.
Trying the above code snippet just gives me a blank web3auth screen/modal though.

<img width="389" alt="Screen Shot 2022-09-13 at 8 22 32" src="https://user-images.githubusercontent.com/112856139/189815735-d72b5d4a-5163-4519-8879-8a0e546139bb.png">

Can someone point me to the right direction with this issue, please?

**Comment by shahbaz17** [comment]
Hey @ealeksandrov7 

It is anwered https://github.com/orgs/Web3Auth/discussions/547#discussioncomment-3632730 by @Tguntenaar

**Reply by ealeksandrov7** [reply]
Thanks!
